{{cookiecutter.python_name}}/{{cookiecutter.python_name}}/__init__.py

```diff
@@ -1,22 +1,12 @@
-import json
-from pathlib import Path
-
 from ._version import __version__{% if cookiecutter.kind.lower() == 'server' %}
 from .handlers import setup_handlers
 {% endif %}
 
 
-HERE = Path(__file__).parent.resolve()
-
-
-with (HERE / "labextension" / "package.json").open() as fid:
-    data = json.load(fid)
-
-
 def _jupyter_labextension_paths():
     return [{
         "src": "labextension",
-        "dest": data["name"]
+        "dest": "{{ cookiecutter.labextension_name }}"
```

Comment:
Is this the cause of the duplication?

Otherwise it could be handy to keep to automatically pick up the changes from `package.json`.

Reply:
No it is not the cause - but if I don't change that I need to include the `package.json` within the Python pkg (aka duplicating it).

Reply:
ah ok then it's fine to drop this, folks will need to remember to this file if they update the name in `package.json` then.